imod/utils.py

```diff
@@ -626,6 +626,18 @@ def generateDefocusIMODFileFromObject(ctfTomoSeries, defocusFilePath,
     else:
         tiltSeries = inputTiltSeries
 
+    # This allows inverting the direction of the defocus gradient if needed.
+    # It does not invert the tilt angles for the reconstruction,
+    # it's only for generating the defocus file!
+    if invertTiltAngles:
+        newTS = tiltSeries.createCopy(outputPath=os.path.dirname(defocusFilePath), 
+                                      prefix="ts_invertedTA", 
+                                      copyInfo=True)
+        for tilt in tiltSeries.iterItems():
+            tilt.setTiltAngle(-1 * tilt.getTiltAngle())
+            newTS.append(tilt)
+            newTS.write()
```

Comment:
I can see you are creating a new sqlite file but I fail to see when/where it is used?

Reply:
You are right, I was doing some tests and forgot to revert to the proper state. It should be good now.
However, now I am not sure that it is working as intended. I will do a few more tests and get back to you. If anybody else can also test I would appreciate it. The goal is just to generate defocus files with inverted tilt angles.